File: AngularMicroservice/src/dto/LabelDTO.ts
import { UserDTO } from './UtenteDTO';

export class LabelDTO{
    idLabel: number;
    name: String;
    idUser: number;

    constructor(id: number, name: String, idUser: number){
        this.idLabel=id;
        this.name=name;
        this.idUser=idUser;
    }
}

File: RobotMicroservice/src/main/java/it/contrader/dto/LabelDTO.java
package it.contrader.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

@Data
@AllArgsConstructor
@NoArgsConstructor
@ToString

public class LabelDTO {
	private Integer idLabel;
	private String name;
	private Integer idUser;
}